snooty/specparser.py

```diff
@@ -262,6 +262,35 @@ def create_role(self) -> Role:
         )
 
 
+@checked
```

Comment:
You only need `@checked` if you're going to be calling `flutter.check_type()`, which I don't see being called in this PR.

Reply:
added the check_type!